### PR TITLE
Add prometheus metrics lists to docs

### PR DESCRIPTION
**Pull request by kentakozuka**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes https://github.com/pipe-cd/pipecd/issues/4516

**Does this PR introduce a user-facing change?**:

- **How are users affected by this change**:
- **Is this breaking change**:
- **How to migrate (if breaking change)**:

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/managing-controlplane/metrics.md [hunk 1]
```diff
@@ -86,3 +86,17 @@ And give it to the `helm install` command when [installing](../../../installatio
 ```
 
 See [here](https://prometheus.io/docs/alerting/latest/configuration/) for more details on AlertManager's configuration.
+
+## Piped agent metrics
+
+| Metric | Type | Description |
+| --- | --- | --- |
+| `deployment_status` | gauge | The current status of deployment. 1 for current status, 0 for others. |
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-controlplane/metrics.md:94]: How about other metrics of piped. We serve other as well
https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/cmd/piped/piped.go#L815-L821

File: docs/content/en/docs-dev/user-guide/managing-controlplane/metrics.md [hunk 2]
```diff
@@ -86,3 +86,17 @@ And give it to the `helm install` command when [installing](../../../installatio
...
+
+## Control plane metrics
+
+All Piped's metrics are sent to the control plane so that they are also available on the control plane's metrics server.
+
+| Metric | Type | Description |
+| --- | --- | --- |
+| `insight_application_total` | gauge | Number of applications currently controlled by control plane. |
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-controlplane/metrics.md:102]: Same as above, we have others metrics of controlplane as well
https://github.com/pipe-cd/pipecd/blob/master/cmd/pipecd/server.go#L512-L518

File: docs/content/en/docs-dev/user-guide/managing-controlplane/metrics.md (hunk 1 above)
Comment (kentakozuka) [reply docs/content/en/docs-dev/user-guide/managing-controlplane/metrics.md:94]: Thanks! I missed them.

**Comment by kentakozuka** [comment]
@khanhtc1202 Other metrics added 👍 
Just thought that this doc is now located under "Managing Control Plane".
How about moving it to under "User Guide" because this doc now explains not only control planes but also piped?

**Comment by khanhtc1202** [comment]
> How about moving it to under "User Guide" because this doc now explains not only control planes but also piped?

That would be great 👍

**Comment by kentakozuka** [comment]
@khanhtc1202 Fixed 👍

**Review by khanhtc1202 — APPROVED** [review]
lgtm ❤️

**Pull request merged**